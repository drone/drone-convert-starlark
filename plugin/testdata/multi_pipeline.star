def main(ctx):
    return [
        {
            "kind": "pipeline",
            "type": "docker",
            "name": "pipeline1",
        },
        {
            "kind": "pipeline",
            "type": "docker",
            "name": "pipeline2",
        },
    ]
