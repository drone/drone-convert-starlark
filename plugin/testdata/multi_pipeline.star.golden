---
{"kind": "pipeline", "type": "docker", "name": "pipeline1"}
---
{"kind": "pipeline", "type": "docker", "name": "pipeline2"}
